typescript-react/src/components/Form/FormSection.module.css

```diff
@@ -49,6 +49,10 @@
   margin-bottom: 16px;
 }
 
+.Section.Titled:not(:last-child) > .Items:empty {
+  margin-bottom: 0;
```

Comment:
This should legit go to upstream, yeah. It can be done via generated global CSS selectors, but is a legit global issue for generated CSS.